charts/rancher-monitoring/104.0.0-rc1+up57.0.3/charts/prometheus-node-exporter/values.yaml

```diff
@@ -6,7 +6,7 @@ image:
   registry: docker.io
   repository: rancher/mirrored-prometheus-node-exporter
   # Overrides the image tag whose default is {{ printf "v%s" .Chart.AppVersion }}
-  tag: v1.3.1
```

Comment:
Remember to add `v1.7.0` to https://github.com/rancher/image-mirror: https://github.com/rancher/image-mirror/blob/master/images-list#L956

Reply:
ope right you are, I think I was looking at the old `prom-node-exporter`, my bad